manager/orchestrator/replicated.go

```diff
@@ -144,6 +139,18 @@ func newTask(cluster *api.Cluster, service *api.Service, slot uint64) *api.Task
 		DesiredState: api.TaskStateRunning,
 		LogDriver:    logDriver,
 	}
+	// We use the following scheme to assign Task names to Annotations:
+	// Annotations.Name := <ServiceAnnotations.Name>.<Slot>.<TaskID> (replicated mode)
+	//                  := <ServiceAnnotations.Name>.<NodeID>.<TaskID> (global mode)
+	if slot != 0 {
```

Comment:
Let's check `if nodeID == ""` instead of `if slot != 0`. Just in case we start using 0 as a valid slot number in the future.